Src/Plugins/Tests.SalModernization/TestData/UpdateSalToCurrentVersion/ExpectedOutputs/SEC105_001.AnalysisAssume.sarif

```diff
@@ -5,63 +5,79 @@
     {
       "tool": {
         "driver": {
-          "name": "testhost",
-          "organization": "Microsoft Corporation",
-          "product": "Microsoft.TestHost",
-          "fullName": "testhost 15.0.0.0",
-          "version": "15.0.0.0",
-          "semanticVersion": "15.0.0",
```

Comment:
>semanticVersion [](http://example.com/codeflow?start=11&length=15)

we no longer persist semantic version data in test logs, as this data contains pre-release information for dev builds that breaks determinism in output. the product will publish this info though.